runtime/lib/capi/CMakeLists.txt

```diff
@@ -30,11 +30,6 @@ target_include_directories(catalyst_qir_qis_obj PUBLIC .
     ${PROJECT_SOURCE_DIR}/../mlir/lib/Driver
     )
 
-# The MLIR Runner Utils raises this warning so we need to disable it for our -Werror builds.
-if(RUNTIME_ENABLE_WARNINGS)
-    target_compile_options(catalyst_qir_qis_obj PRIVATE "-Wno-unused-parameter")
-endif()
```

Comment:
Oh this is the Catalyst specific flag `RUNTIME_ENABLE_WARNINGS` (as opposed to the lightning flag `ENABLE_WARNINGS`). I think we should keep it.

Reply:
Restored